--- lib3ds-1.2.0/configure.in.orig	Mon Jan 14 13:09:15 2002
+++ lib3ds-1.2.0/configure.in	Mon Mar 24 20:11:14 2003
@@ -38,7 +38,7 @@
 
 dnl Checks for programs.
 AC_PROG_CC
-AC_PROG_RANLIB
+AC_PROG_LIBTOOL
 CONFIGURE_GLUT(,)
 
 AC_MSG_CHECKING([for debugging mode])
--- lib3ds-1.2.0/lib3ds/Makefile.am.orig	Mon Jan 14 13:27:45 2002
+++ lib3ds-1.2.0/lib3ds/Makefile.am	Mon Mar 24 20:10:12 2003
@@ -24,9 +24,9 @@
 
 INCLUDES = -I$(top_srcdir)
 
-lib_LIBRARIES = lib3ds.a
+lib_LTLIBRARIES = lib3ds.la
 
-lib3ds_a_SOURCES = \
+lib3ds_la_SOURCES = \
   io.c \
   float.c \
   vector.c \
@@ -48,6 +48,8 @@
   tracks.c \
   node.c
 
+lib3ds_la_LIBADD = -lm
+
 lib3ds_HEADERS = \
   types.h \
   io.h \
--- lib3ds-1.2.0/examples/Makefile.am.orig	Mon Jan 14 13:27:45 2002
+++ lib3ds-1.2.0/examples/Makefile.am	Mon Mar 24 20:10:47 2003
@@ -36,12 +36,12 @@
   $(PLAYER)
 
 LDADD = \
-  $(top_builddir)/lib3ds/lib3ds.a \
+  $(top_builddir)/lib3ds/lib3ds.la \
   @DMALLOC_LIBS@ -lm
 
 player_SOURCES = player.c
 player_LDADD = \
-  $(top_builddir)/lib3ds/lib3ds.a \
+  $(top_builddir)/lib3ds/lib3ds.la \
   @GLUT_LIBS@ \
   @DMALLOC_LIBS@ -lm
 
--- lib3ds-1.2.0/tools/Makefile.am.orig	Mon Jan 14 13:27:45 2002
+++ lib3ds-1.2.0/tools/Makefile.am	Mon Mar 24 20:22:08 2003
@@ -29,7 +29,7 @@
   3ds2m 
 
 LDADD = \
-  $(top_builddir)/lib3ds/lib3ds.a \
+  $(top_builddir)/lib3ds/lib3ds.la \
   @DMALLOC_LIBS@ -lm
 
 MANPAGES = \
